scripts/alertmailer.php

```diff
@@ -237,25 +255,45 @@ function mlog($message) {
                 ':time' => date('Y-m-d H:i:s', $lastupdated),
             ));
         foreach ($q as $row) {
+            # Skip other-language divisions if needed, set locale
+            if (strpos($row['division_id'], '-cy-')) {
+                if ($lang == 'en') {
+                    continue;
+                }
+                change_locale('cy');
+            } elseif (strpos($row['division_id'], '-en-')) {
+                if ($lang == 'cy') {
+                    continue;
+                }
+                change_locale('en');
+            } else {
+                change_locale('en');
+            }
+
             $vote = $row['vote'];
             $num = $row['division_number'];
             $teller = '';
             if (strpos($vote, 'tell') !== false) {
-                $teller = ', as a teller';
+                $teller = ', ' . gettext('as a teller');
                 $vote = str_replace('tell', '', $vote);
             }
             if ($vote == 'absent') {
                 continue;
             }
-            $text = "Voted ";
             if ($vote == 'aye' && $row['yes_text']) {
-                $text .= "($vote$teller) " . $row['yes_text'];
+                $text = "Voted ($vote$teller) " . $row['yes_text'];
```

Comment:
I think $teller can sometimes be blank; it's more because the yes_text/no_text has no way of being translated, so I assume we won't currently be using this (it's for the UK override better-description text we occasionally use).